frontend/components/bc/BcContentFilter.vue

```diff
@@ -1,108 +0,0 @@
-<script lang="ts" setup>
```

Comment:
Why you renamed it again? Didn't you just recently add the Bc?  
I made a style change in this file by adding the `data-secondary` attribute on the Button. This is missing in the other file

Reply:
damn...merge conflict stuff...suddenly there where two files

Reply:
copied over the staging version